src/NativeFunctionCall.ts

```diff
@@ -129,7 +129,8 @@ export class NativeFunctionCall extends InkObject{
 			if (this._operationFuncs === null) return throwNullException('NativeFunctionCall._operationFuncs');
 			let opForTypeObj = this._operationFuncs.get(valType);
 			if (!opForTypeObj) {
-				throw new StoryException('Cannot perform operation '+this.name+' on '+valType);
+        const key = Object.keys(ValueType)[valType];
```

Comment:
Wouldn't `ValueType[valType]` work just as well? It's a bit more TypeScript-y, but I haven't tested it. 🙃